Gopkg.toml

```diff
@@ -5,7 +5,7 @@
 
 [[constraint]]
   name = "github.com/container-storage-interface/spec"
-  version = "0.1.0"
+  revision = "7ab01a90da87f9fef3ee1de0494962fdefaf7db7"
```

Comment:
Hi @sbezverk,

Tagging a specific revision in `Gopkg.toml` should almost always be avoided unless overriding a transitive constraint. Please change this to `branch = "master"` and then run `dep ensure` to regenerate the `Gopkg.lock` file.

Reply:
maybe there is a bug in dep, but when I use ```branch = "master"``` vendor files do not get updated.

Reply:
Hi @sbezverk,

Which vendor files? It's unclear to me that we're on the same page regarding dep and vendor.

Reply:
Hello @akutz sorry I am sure I have missed something that is why all these back and forth. At one point, I thought that you do not allow PR to carry its vendor folder which was generated by running ```dep ensure``` on the local machine. Please confirm if I can submit as a second commit of this PR my vendor folder.

Reply:
Hi @sbezverk,

I'm going to respond to this above, in our longer conversation.